src/redemptions/redemption.service.ts

```diff
@@ -138,6 +150,59 @@ export class RedemptionService {
     return null;
   }
 
+  async repeatedFaceCheck(
+    response: JumioTransactionRetrieveResponse,
+  ): Promise<{ status: KycStatus; failureMessage: string } | null> {
+    // can only accept if no other errors occurred AND repeated face is from previous transaction of SAME user
+    let potentialFailures: string[] = [];
+    potentialFailures = potentialFailures.concat(
+      response.capabilities.dataChecks.map((i) => i.decision.type),
+      response.capabilities.extraction.map((i) => i.decision.type),
+      response.capabilities.liveness.map((i) => i.decision.type),
+      response.capabilities.similarity.map((i) => i.decision.type),
+      response.capabilities.usability.map((i) => i.decision.type),
+    );
+    // if any other check failed, we can't pass
+    for (const potentialFailure of potentialFailures) {
+      if (potentialFailure !== 'PASSED') {
+        return null;
+      }
+    }
```

Comment:
I think you can delete everything above this point, and just continue execution of the current flow instead of early return